frontend/cypress/integration/featureFiles/graph_manipulation.feature

```diff
@@ -0,0 +1,140 @@
+Feature: Kiali Graph page - Graph manipulation 
+
+  User opens the Graph page and manipulates the "alpha", "beta" namespace graph with buttons
+    located at the bottom of the page 
+
+  Background:
+    Given user is at administrator perspective
+    And user graphs "alpha,beta" namespaces
+
+@graph-page-manipulation
+Scenario: Check if all the buttons are usable
+  Then the toggle button "Toggle Drag" is enabled
+  And the toggle button "Zoom to Fit" is enabled
+  And the toggle button "Hide Healthy Edges" is enabled
+  And the toggle button "Hide All Edges" is enabled
+  And the toggle button "Graph Layout Default Style" is enabled
+  And the toggle button "Graph Layout Style 1" is enabled
+  And the toggle button "Graph Layout Style 2" is enabled
+  And the toggle button "Graph Layout Style 3" is enabled
+  And the toggle button "Namespace Layout Style 1" is enabled
+  And the toggle button "Namespace Layout Style 2" is enabled
+  And the toggle button "Show Legend" is enabled
```

Comment:
This seems like it would be a good table test similar to the `Scenario Outline` below.